modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/java/com/liferay/portal/search/tuning/rankings/web/internal/util/RankingResultUtil.java

```diff
@@ -87,7 +86,7 @@ public static String getRankingResultViewURL(
 			_getSearchResultInterpreter();
 
 		PortletURL viewContentURL = resourceResponse.createRenderURL();
-		String currentURL = _portal.getCurrentURL(resourceRequest);
+		String currentURL = PortalUtil.getCurrentURL(resourceRequest);
```

Comment:
@slnn Please modify the commit message to say that you do this because you are going to make `RankingResultUtil` a pure util.